requirements.txt

```diff
@@ -1,7 +1,5 @@
-nowcasting_utils
-PVNet-summation>=0.0.7
-nowcasting_datamodel==1.5.10
 ocf_datapipes==1.2.44
+nowcasting_utils
```

Comment:
be nice to to include this? whats it needed for?

Reply:
Its for some of the metrics. The exponential weighted loss and the forecast mae at each time horizon. I could pull those out and put them in PVNet if we want to remove the import?

Reply:
lhmm, thats ok, leave it